### PR TITLE
Emails users about shutoff VMs

### PR DESCRIPTION
**Pull request by AkhilMag13**
### Description:
Added an action for users to get an email when they have VMs in shutoff state

---

### Submitter:

Have you (where applicable):

* [x] Added unit tests?
* [ ] Checked the latest commit runs on Dev?
* [ ] Updated the example config file(s) and README?

---

### Reviewer

Does this PR:

* [ ] Place non-StackStorm code into the `lib` directory?
* [ ] Have unit tests for the action/sensor and `lib` layers?
* [ ] Have clear and obvious action parameter names and descriptions?

### CODE REVIEW

File: lib/workflows/send_shutoff_vm_email.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+        else:
+            email_params = build_email_params(
+                user_name,
+                server_list,
+                email_to=send_to,
+                as_html=as_html,
+                email_cc=("cloud-support@stfc.ac.uk",) if cc_cloud_support else None,
+                **email_params_kwargs,
+            )
+            Emailer(smtp_account).send_emails([email_params])
```
Comment (anish-mudaraddi) [inline lib/workflows/send_shutoff_vm_email.py:174]: NIT: remove unnecessary else by returning early

```suggestion
        if not send_email:
            print_email_params(send_to[0], user_name, as_html, server_list)
            return
        
        email_params = build_email_params(
            user_name,
            server_list,
            email_to=send_to,
            as_html=as_html,
            email_cc=("cloud-support@stfc.ac.uk",) if cc_cloud_support else None,
            **email_params_kwargs,
        )
        Emailer(smtp_account).send_emails([email_params])
```

File: lib/workflows/send_shutoff_vm_email.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    """
+
+    # Find VMs that have been in shutoff state for more than 60 days
+    server_query = ServerQuery()
+    server_query.where(
+        QueryPresetsGeneric.ANY_IN, ServerProperties.SERVER_STATUS, values=["SHUTOFF"]
+    )
+    server_query.run(
+        cloud_account,
+        as_admin=False,
```
Comment (anish-mudaraddi) [inline lib/workflows/send_shutoff_vm_email.py:30]: change this to `True` otherwise it won't work for all projects!
You'll need to adjust the tests too!

**Review by anish-mudaraddi — CHANGES_REQUESTED** [review]
small changes - otherwise LGTM

File: lib/workflows/send_shutoff_vm_email.py (hunk 1 above)
Comment (AkhilMag13) [reply lib/workflows/send_shutoff_vm_email.py]: This gives an error as if there are multiple calls being made, it returns after the first call as can be seen in the test failing in [dbf8dbb](https://github.com/stfc/st2-cloud-pack/pull/234/commits/dbf8dbbf26170e73359102d52d84c80efc08c96e)

File: lib/workflows/send_shutoff_vm_email.py (hunk 2 above)
Comment (AkhilMag13) [reply lib/workflows/send_shutoff_vm_email.py]: Thank you, fixed this in [dbf8dbb](https://github.com/stfc/st2-cloud-pack/pull/234/commits/dbf8dbbf26170e73359102d52d84c80efc08c96e)

File: lib/workflows/send_shutoff_vm_email.py (hunk 1 above)
Comment (anish-mudaraddi) [reply lib/workflows/send_shutoff_vm_email.py]: change it to `continue` - forgot we were in a loop

Comment (AkhilMag13) [reply lib/workflows/send_shutoff_vm_email.py]: Thanks, fixed it with [c1a2837](https://github.com/stfc/st2-cloud-pack/pull/234/commits/c1a28373649d9e8eaf447e15fac781414a079e52)

**Review by anish-mudaraddi — APPROVED** [review]

**Review by RWhiting78 — APPROVED** [review]

**Pull request merged**